### PR TITLE
[Tests-Only] Refactor `webUIResharing` and `webUIResharingToRoot` tests to not use skeleton files

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Refactor  `webUIResharing` and `webUIResharingToRoot`  tests to not use skeleton files:
- webUIResharing1
- webUIResharing2
- webUIResharingToRoot

## Related Issue
https://github.com/owncloud/QA/issues/659

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature [hunk 1]
```diff
@@ -181,73 +187,74 @@ Feature: Resharing shared files with different permissions
...
-    And user "Carol" has shared folder "simple-folder (2)" with group "Davidgrp"
+    And user "Brian" has shared folder "simple-folder" with user "Alice"
+    And user "Brian" has shared folder "simple-folder" with user "Carol"
+    And user "Alice" has shared folder "simple-folder" with user "David"
+    And user "Carol" has shared folder "simple-folder" with group "Davidgrp"
     When user "David" has logged in using the webUI
-    And the user opens the share dialog for folder "simple-folder (2)" using the webUI
-    Then user "Alice Hansen" should be listed as "Owner" reshared through "Brian Murphy, Carol King" in the collaborators list on the webUI
+    And the user opens the share dialog for folder "simple-folder" using the webUI
+    Then user "Brian Murphy" should be listed as "Owner" reshared through "Alice Hansen, Carol King" in the collaborators list on the webUI
```
Comment (Talank) [inline tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:260]: Maybe you don't need to change the owner from `Alice` to `Brian` 🤔 Other than that, everything looks good to me

Comment (saw-jan) [reply tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:260]: In `Background` I have created a folder for user `Brian` as  Brian is used to share resource in most of the tests there, so in order to use that I have changed some tests that use `Alice` to share resource.
That way, we can omit creating files folders for user Alice also

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14541/

<p>




</p></details>

File: tests/acceptance/features/webUIResharing1/reshareUsers.feature [hunk 2]
```diff
@@ -103,32 +104,42 @@ Feature: Resharing shared files with different permissions
...
-      | shared-entry-name |
-      | simple-folder     |
-      | lorem.txt         |
-
+    Then the user should not be able to share resource "simple-folder" using the webUI
+  
+  @issue-product-270
+  Scenario: share a file without share permissions and check if another user can reshare
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "lorem.txt" with user "Alice" with "read" permissions
```
Comment (swoichha) [inline tests/acceptance/features/webUIResharing1/reshareUsers.feature:117]:
```suggestion
    And user "Brian" has shared file "lorem.txt" with user "Alice" with "read" permissions
```

File: tests/acceptance/features/webUIResharing1/reshareUsers.feature [hunk 3]
```diff
@@ -103,32 +104,42 @@ Feature: Resharing shared files with different permissions
...
 
   @issue-product-270
-  Scenario Outline: share a received file/folder without share permissions and check if another user can reshare
-    Given user "Brian" has shared folder "<shared-entry-name>" with user "Alice" with "all" permissions
-    And user "Alice" has accepted the share "<shared-entry-name>" offered by user "Brian"
-    And user "Alice" has shared folder "/Shares/<shared-entry-name>" with user "Carol" with "read" permissions
-    And user "Carol" has accepted the share "<shared-entry-name>" offered by user "Alice"
+  Scenario: share a received file without share permissions and check if another user can reshare
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "lorem.txt" with user "Alice" with "all" permissions
```
Comment (swoichha) [inline tests/acceptance/features/webUIResharing1/reshareUsers.feature:136]:
```suggestion
    And user "Brian" has shared file "lorem.txt" with user "Alice" with "all" permissions
```

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature [hunk 4]
```diff
@@ -103,75 +104,80 @@ Feature: Resharing shared files with different permissions
...
-    Examples:
-      | shared-entry-name | received-entry-name |
-      | simple-folder     | simple-folder (2)   |
-      | lorem.txt         | lorem (2).txt       |
+    Then the user should not be able to share resource "simple-folder" using the webUI
 
 
+  Scenario: share a file without share permissions and check if another user can reshare
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "lorem.txt" with user "Alice" with "read" permissions
```
Comment (swoichha) [inline tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:140]: same as above

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature [hunk 5]
```diff
@@ -103,75 +104,80 @@ Feature: Resharing shared files with different permissions
...
-      | shared-entry-name | received-entry-name |
-      | simple-folder     | simple-folder (2)   |
-      | lorem.txt         | lorem (2).txt       |
+    Then the user should not be able to share resource "simple-folder" using the webUI
+
+
+  Scenario: share a received file without share permissions and check if another user can reshare
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "lorem.txt" with user "Alice" with "all" permissions
+    And user "Alice" has shared folder "lorem.txt" with user "Carol" with "read" permissions
```
Comment (swoichha) [inline tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:155]: same as above

**Review by swoichha — CHANGES_REQUESTED** [review]

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14547/

<p>




</p></details>

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature [hunk 6]
```diff
@@ -103,75 +104,80 @@ Feature: Resharing shared files with different permissions
...
-      | shared-entry-name | received-entry-name |
-      | simple-folder     | simple-folder (2)   |
-      | lorem.txt         | lorem (2).txt       |
+    Then the user should not be able to share resource "simple-folder" using the webUI
+
+
+  Scenario: share a received file without share permissions and check if another user can reshare
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared file "lorem.txt" with user "Alice" with "all" permissions
+    And user "Alice" has shared folder "lorem.txt" with user "Carol" with "read" permissions
```
Comment (swoichha) [inline tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:155]:
```suggestion
    And user "Alice" has shared file "lorem.txt" with user "Carol" with "read" permissions
```

File: tests/acceptance/features/webUIResharing1/reshareUsers.feature [hunk 7]
```diff
@@ -185,77 +196,79 @@ Feature: Resharing shared files with different permissions
 
 
   Scenario: Reshare a file and folder from shared with me page
-    Given user "Alice" has shared folder "simple-folder" with user "Brian"
-    And user "Brian" has accepted the share "simple-folder" offered by user "Alice"
-    And user "Alice" has shared file "lorem.txt" with user "Brian"
-    And user "Brian" has accepted the share "lorem.txt" offered by user "Alice"
-    And user "Brian" has logged in using the webUI
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "simple-folder" with user "Alice"
```
Comment (individual-it) [inline tests/acceptance/features/webUIResharing1/reshareUsers.feature:200]: why switching the users around?

Comment (saw-jan) [reply tests/acceptance/features/webUIResharing1/reshareUsers.feature:200]: because https://github.com/owncloud/web/pull/4953#discussion_r611512523

**Review by individual-it — APPROVED** [review]

**Pull request merged**